read-fonts/src/lib.rs

```diff
@@ -71,6 +71,86 @@ pub(crate) mod codegen_prelude {
 
 include!("../generated/font.rs");
 
+#[derive(Clone)]
+/// Reference to the content of a font or font collection file.
+pub enum FileRef<'a> {
+    /// A single font.
+    Font(FontRef<'a>),
+    /// A collection of fonts.
+    Collection(CollectionRef<'a>),
+}
+
+impl<'a> FileRef<'a> {
+    pub fn new(data: FontData<'a>) -> Result<Self, ReadError> {
```

Comment:
I think we should start adopting the policy of documenting all public API.

Reply:
👍